### PR TITLE
i1849: New SSL certificate

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1849

### CODE REVIEW

File: certs/production/montagu.vaccineimpact.org.crt [hunk 1]
```diff
@@ -1,110 +1,39 @@
 -----BEGIN CERTIFICATE-----
-MIIFVzCCBD+gAwIBAgIJAN/wllrH5ZvyMA0GCSqGSIb3DQEBCwUAMIG0MQswCQYD
```
Comment (littledice) [inline certs/production/montagu.vaccineimpact.org.crt]: Why did this have 3 certificates before, and now just 1?

Comment (MartinEden) [reply certs/production/montagu.vaccineimpact.org.crt]: Very good catch. It turns out the thing I was using to inspect the certificate file was only printing out the first certificate in the bundle, leading me to think it wasn't a bundle. I've improved that now, and have (I think) properly created a new bundle.

**Comment by MartinEden** [comment]
We don't need a new certificate for support - that one's internal, provided by Imperial (not GoDaddy) and lasts until 2020.

**Comment by MartinEden** [comment]
Here is the diff between the old certificate bundle and new certificate bundle in human-readable form:
[diff.txt](https://github.com/vimc/montagu/files/2132640/diff.txt)

**Pull request merged**